docs/nontech-explainer.md

```diff
@@ -0,0 +1,65 @@
+
+## How To Make Money in The Open
+
+Now, all companies need to make money of course. For that, we need to answer the needs of our customers. And in a constantly changing world, knowing what the public actually wants is a real challenge[^3]. Building a successful product requires extensive surveys and tests (aka _"crowdsourcing"_) to improve market fit.
+
+Openness provides a framework to lessen these needs considerably. By building a project and company with constant contributions from outsiders, we are guaranteed to receive most feedback we need without any additional effort or spending. Better yet, Openness build strong long-terme genuine relations, where crowdsourcing only relies on short-term self-serving requests.
+
+But this doesn't answer the main question all people have when talking about Open Businesses: how can we make money if we share our product with everyone? After all, companies like Louis Vuitton don't provide Open Specifications to show how to build my own luxury bag. There should be a good reason behind that, right?
+
+Right. And this reason's called branding. Luxury brands like Louis Vuitton need to guarantee a certain level of quality for their products, because it's linked to their brand image. Controlling manufacture is their only option[^4].
+
+Openness in technology is the opposite. First, because the software is, by definition, immaterial. Copying it doesn't lessen its quality. And second because by adopting Openness principles, you can prove to your customers your mastery of the related topics.
+
+Here, again, Open Source isn't enough by itself. We need to nurture a constant collaboration flow with the community to guarantee we have a better brand image and innovate faster than the competition.
+
+Based on this image and constant innovation, a company can offer a large set of offerings _around_ its Open Sourced productions:
+
+- Software As a Service, to make the software easier to use, more stable, more efficient, etc.,
+- proprietary (i.e. "closed") extensions on top of an Open "core" to answer more specific needs,
+- Professionnal Services, like consulting, support and training,
+- and more.
+
+## Why We Can't Afford Closeness
+
+Some companies can still manage to make money by selling proprietary software.
+
+First, there are Big Corps, like Microsoft or Oracle. Their first common denominator is a very strong and well-established brand. People can trust them because of this brand image, without needing to collaborate with them actively[^5]. Today, blindnet doesn't fit in this category. We are the new challenger who's here to disrupt the market with innovation and an outsider's perspective. We need to use weapons that fit this challenge.
+
+Then, there are the products for non-technical people, on which the brand image and market fit can be built with traditional marketings tools like ads and surveys.
```

Comment:
I am not sure this statement makes concensus :) i.e. that Microsoft and Oracle would agree.

Some of our, ex-Microsoft, investors might raise an eybrow.

Reply:
Now that you said it, I realize I don't even agree with this myself.
Let me know if this new sentence is good enough (and grammatically correct).